cats/__init__.py

```diff
@@ -48,24 +54,41 @@ def parse_arguments():
        cats -d 1 --loc RG1 --scheduler=at --command='ls'
     """
 
-    example_text = """
-    Examples\n
-    ********\n
+    config_text = indent_lines(
+        Path(__file__).with_name("config.yml").read_text(), spaces=8
+    )
+    example_text = f"""
+    Examples
+    ********
 
-    Cats can be used to report information on the best time to run a calculation and the amount
-    of CO2. Information about a 90 minute calculation in centeral Oxford can be found by running:
+    CATS can be used to report information on the best time to run a calculation
+    and the amount of CO2. Information about a 90 minute calculation in centeral
+    Oxford can be found by running:
 
-        cats -d 90 --loc OX1 --jobinfo="cpus=2,gpus=0,memory=8,partition=CPU_partition"
```

Comment:
Oops, I forgot to remove this reference to the old 'jobinfo' syntax. Good catch :)